resources/migrations/058_update_migrations.yaml

```diff
@@ -835,6 +835,77 @@ databaseChangeLog:
             tableName: segment
             columnName: dependency_analysis_version
 
+  - changeSet:
+      id: v58.2025-11-25T17:41:50
+      author: metamben
+      comment: Add collection_id column to metabase_table for publishing tables to collections
```

Comment:
We're not doing the MtM table anymore? Are we 100% sure we're never going to need to put a table in more than one collection, ever? We'd never have a situation where we want to put customer_issues in both the Eng dashboard and Success dashboard?

Reply:
We are not doing that. It was deemed too complicated reason about permissions with multiple paths.